tests/www/TemplateTest.php

```diff
@@ -1,69 +0,0 @@
-<?php
```

Comment:
This one is removed because we now have syntax checkers for twig templates directly in the build, right?